schema/compose-spec.json

```diff
@@ -8,7 +8,7 @@
   "properties": {
     "version": {
       "type": "string",
-      "description": "declared for backward compatibility, ignored."
+      "description": "The version of the Docker Compose document"
```

Comment:
Please, keep a warning so user **don't** use this anymore

Reply:
Sure thing, updated

Reply:
Sure thing, updated